### PR TITLE
Add increase budget UI

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
QA: there's currently no way to QA the deployment flow.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5647/
Test PASSed.

File: jujugui/static/gui/src/app/components/deployment-flow/budget/budget.js [hunk 1]
```diff
@@ -145,14 +162,67 @@ YUI.add('deployment-budget', function() {
...
+              </div>
+              <div className="three-col last-col">
+                <juju.components.GenericInput
+                  disabled={true}
+                  label="New budget amount"
+                  required={false} />
+              </div>
+              <div>
+                <div className="eight-col">
+                <span className="link">Manage all budgets</span>
```
Comment (anthonydillon) [inline jujugui/static/gui/src/app/components/deployment-flow/budget/budget.js]: Missing indentation

**Comment by anthonydillon** [comment]
Looks good :+1: some trivial comments in code.

**Comment by hatched** [comment]
Looks good, thanks @huwshimi

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5656/
Test PASSed.

**Comment by huwshimi** [comment]
Thanks @anthonydillon, @hatched :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**